src/components/messages/connected-user-message.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from "react";
+import { Message } from "../../models";
+import { StyleSheet, Text, View } from "react-native";
+import { getTime } from "../../helpers/messages/get-time";
+
+const styles = StyleSheet.create({
+  connectedUserContainer: {
+    flex: 1,
+    paddingHorizontal: 10,
+    marginVertical: 5,
+    alignItems: "flex-end",
+  },
+  otherUserContainer: {
+    flex: 1,
+    paddingHorizontal: 10,
+    marginVertical: 5,
+    alignItems: "flex-start",
+  },
+  connectedUserMessageContainer: {
+    paddingVertical: 7,
+    paddingHorizontal: 10,
+    backgroundColor: "#2196F3",
+    borderRadius: 20,
+  },
+  otherUserMessageContainer: {
+    paddingVertical: 7,
+    paddingHorizontal: 10,
+    backgroundColor: "#ECECEC",
+    borderRadius: 20,
+  },
+  text: {
+    color: "white",
+    fontSize: 17,
+  },
+  date: {
+    color: "#9E9E9E",
+    fontSize: 12,
+  },
+});
+
+type Props = {
+  message: Message;
+  sender: boolean;
+};
+
+export const ConnectedUserMessage: FC<Props> = (props: Props) => {
```

Comment:
Need to change this component in "UserMessage" instead of "ConnectedUserMessage", same for the file